codemods/babel-plugin-codemod-object-assign-to-object-spread/test/index.js

```diff
@@ -1,3 +1,5 @@
 import runner from "@babel/helper-plugin-test-runner";
+import { fileURLToPath } from "url";
+import { dirname } from "path";
 
-runner(__dirname);
+runner(dirname(fileURLToPath(import.meta.url)));
```

Comment:
Ideally this could become `runner(import.meta.url)`, but I'd like to avoid runtime changes in this PR.